posthog/models/web_preaggregated/sql.py

```diff
@@ -57,9 +57,11 @@ def HOURLY_TABLE_TEMPLATE(table_name, columns, order_by, ttl=None):
     "city_name",
     "region_code",
     "region_name",
+    "has_gclid",
```

Comment:
Nice!

In the future we could consider doing something like "has_paid_id", "has_social_id", etc, which would stop these from exploding if we supported more, but you don't need to worry about this for now